components/03-forms/01-groups/group-files.njk

```diff
@@ -0,0 +1,37 @@
+<div {{ className('form-group', 'dso-files', [required, 'dso-required']) }}>
+  <div class="dso-description dso-rich-content">
+    {{ description }}
+  </div>
+  <ul class="dso-filelist">
+    {% for file in files %}
+      {% render '@file-item', {filename: file.filename, pos: loop.index0, confidential: file.confidential} %}
+    {% endfor %}
+  </ul>
+  {% for file in files %}
+    {% if file.confidential %}
+      {% render '@alert', {modifier: 'warning', message:'
+        <div class="dso-rich-content">
+          <p>U vraagt of wij een document vertrouwelijk willen behandelen. Hiervoor moet u twee dingen doen:</p>
+          <ol>
+            <li>Voeg ook een publiceerbaar document zonder vertrouwelijke informatie toe.</li>
+            <li>Geef per document de reden op als toelichting op het verzoek in de laatste stap van uw aanvraag.</li>
+          </ol>
+          <p>Er zal worden beoordeeld of uw vraag terecht is. U krijgt hiervan bericht.</p>
+        </div>
+      '} %}
+    {% endif %}
+  {% endfor %}
+
+  {# {# <!-- To use an input type="file" element --> #}
+  <div class="dso-file-upload">
+    <input type="file" id="file-upload" />
+    <label class="btn btn-link" for="file-upload">
+      {% render '@icon' , {icon: 'plus'} -%}<span>Document toevoegen</span>
+    </label>
+  </div>
+
+  <!-- Or to trigger the upload using a button element (not recommended) -->
+  <!--
+    {% render '@button', {label: 'Document toevoegen', modifier: 'link', type: 'button', icon: 'plus'}, true %}
```

Comment:
Waarom staat hier `true` voor het mergen? In de oorspronkelijke handlebars is geen `merge=true`

Reply:
Ik haal 'm weg, ik denk dat het een copy/paste foutje is. Volgens `npm run gulp test` issie ook niet nodig